.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: false
+language: go
+
+go:
+  - 1.7.x
+  - 1.8.x
+  - tip
+
+script:
+  - make build
```

Comment:
Tests are not even compiling in this branch 🤦‍♂️. They are already fixed in `develop` branch. The idea of that PR is to make exporter working and `go get`-able ASAP.